repository_service_tuf/cli/admin/metadata.py

```diff
@@ -709,18 +706,16 @@ def _sign_metadata(role_info: MetadataInfo, rstuf_key: RSTUFKey) -> Signature:
 def _delete_signing_process(
     role: str, settings: Any, api_url: Optional[str]
 ) -> None:
-    if settings.AUTH is False and api_url is None:
+    if api_url is None:
         api_url = prompt.Prompt.ask("\n[cyan]API[/] URL address")
```

Comment:
Why do you request a user to enter `API URL` if he has already setup `settings.SERVER`? 
Also, `_delete_signing_process` is called from the `sign` command where `api_url` is optional, but how can it be optional where we check if it's `None` and if it is we require value?